src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -775,12 +799,17 @@ private UsingDirectiveSyntax ParseUsingDirective()
                 semicolon = this.EatToken(SyntaxKind.SemicolonToken);
             }
 
-            var usingDirective = _syntaxFactory.UsingDirective(usingToken, staticToken, alias, name, semicolon);
+            var usingDirective = _syntaxFactory.UsingDirective(globalToken, usingToken, staticToken, alias, name, semicolon);
             if (staticToken != null)
             {
                 usingDirective = CheckFeatureAvailability(usingDirective, MessageID.IDS_FeatureUsingStatic);
             }
 
+            if (globalToken != null)
+            {
+                usingDirective = CheckFeatureAvailability(usingDirective, MessageID.IDS_FeatureUsingGlobal);
```

Comment:
nit: why put the diagnostic on the whole directive? I expected we'd check the LangVer when we eat the token above (line 756) and put the diagnostic on `globalToken` (it's a little more specific)

Reply:
> nit: why put the diagnostic on the whole directive? I expected we'd check the LangVer when we eat the token above (line 756) and put the diagnostic on globalToken (it's a little more specific)

I am following the pattern already established for the `static` keyword.

---
In reply to: [581664800](https://github.com/dotnet/roslyn/pull/51426#discussion_r581664800) [](ancestors = 581664800)